main/test/flix/Test.Exp.Jvm.NewObject.flix

```diff
@@ -1,32 +1,11 @@
 import java.lang.Object
-import flix.test.TestBoolArrayInterface
-import flix.test.TestBoolInterface
-import flix.test.TestCharArrayInterface
-import flix.test.TestCharInterface
-import flix.test.TestClass
-import flix.test.TestClassWithDefaultConstructor
-import flix.test.TestClassWithNonDefaultConstructor
-import flix.test.TestDefaultMethods
-import flix.test.TestFloat32ArrayInterface
-import flix.test.TestFloat32Interface
-import flix.test.TestFloat64ArrayInterface
-import flix.test.TestFloat64Interface
-import flix.test.TestFunctionalInterface
-import flix.test.TestGenericInterface
-import flix.test.TestGenericMethod
-import flix.test.TestInt8ArrayInterface
-import flix.test.TestInt16ArrayInterface
-import flix.test.TestInt16Interface
-import flix.test.TestInt32ArrayInterface
-import flix.test.TestInt32Interface
-import flix.test.TestInt64ArrayInterface
-import flix.test.TestInt64Interface
-import flix.test.TestNonPublicInterface
-import flix.test.TestOverloadedMethods
-import flix.test.TestStackOffsets
-import flix.test.TestThrowingInterface
-import flix.test.TestVarargsInterface
-import flix.test.TestVoidInterface
+import flix.test.{TestBoolArrayInterface, TestBoolInterface, TestCharArrayInterface, TestCharInterface, 
```

Comment:
Ideally we would want test cases for both import one and import many, but maybe we already have that :)